src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -32,20 +34,43 @@ spec:
   displayName: MyOnlineShop
   rbac:
     admins:
-    - platform-team
+    - customer:platform-admin
+    - customer:ops-team
     editors:
-    - development-team
+    - customer:development-team
+    - customer:devops-team
     viewers:
-    - marketing-team
+    - customer:marketing-team
+    - customer:business-analysts
   tenants:
   - myonlineshop
 ```
 
-Our operators will create this `Grafana` organization named _Giant Swarm_. It will be equipped with a basic set of data sources for Loki, Mimir and Alertmanager, allowing you access to the `myonlineshop` tenant.
+Our operators will create this `Grafana` organization named _MyOnlineShop_. The system equips it with a basic set of data sources for Loki, Mimir and Alertmanager, giving you access to the `myonlineshop` tenant.
+
+## RBAC configuration
+
+The Role Based Access Control (RBAC) section defines how to assign groups from your configured identity provider to `Grafana` [organization roles](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#organization-roles) (`Admin`, `Editor`, `Viewer`).
+
+### Dex connector format
+
+Most Giant Swarm installations use Dex as the identity provider. When using Dex, specify groups using the format `{dex-connector-id}:{group-name}`. For example:
+
+- `customer:platform-admin` - Maps the `platform-admin` group from the `customer` Dex connector to Grafana Admin role
+- `customer:development-team` - Maps the `development-team` group from the `customer` Dex connector to Grafana Editor role
+
+You can find your Dex connector ID in your cluster's Dex configuration. Only the `admins` field is mandatory in the RBAC section.
+
+For more details about organization mapping, see the official Grafana [OAuth documentation](https://grafana.com/docs/grafana/next/setup-grafana/configure-security/configure-authentication/generic-oauth/#configure-role-mapping).
+
+## Tenant governance
+
+The tenant field grants access to specified tenants and serves as tenant governance. The system only accepts tenants listed in **at least one** Grafana Organisation CRD as valid targets in the write path.
```

Comment:
Maybe we should explain what a `tenant` is.
I feel like the relationship between `RBAC group`, `Organization` and `tenant` can be hard to understand.
Probably a drawing would help.
Something that would show:
- that you collect data on your clusters, and push it to some databases (mimir / loki) with an associated tenant
- and that the grafana organization creates a link between oauth groups and data tenants
- (and maybe explain at some point that allowing multiple tenants for 1 organization has some limitations)

Reply:
I agree that it makes sense. What do you think about adding it in another PR though? It would probably be a good pairing topic